### PR TITLE
Add node lease namespace

### PR DESCRIPTION
**Pull request by cloudmelon**
add node lease namespace

### CODE REVIEW

**Comment by sftim** [comment]
Thanks @cloudmelon 
/sig node

**Comment by cloudmelon** [comment]
@kbhawkey  It's been quite a while my PR ( I reopen this one yday ), can you have a look this PR and should be OK to merge since we had many back and forth at a time.

**Comment by kbhawkey** [comment]
@wojtek-t , Do you have time to review this change for technical accuracy (Lease object)?  Thanks!

**Comment by sftim** [comment]
/retitle Add node lease namespace
copying from PR #20082

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -58,12 +58,13 @@ kube-public       Active   1d
...
 ```
 
-Kubernetes starts with three initial namespaces:
+Kubernetes starts with four initial namespaces:
 
    * `default` The default namespace for objects with no other namespace
    * `kube-system` The namespace for objects created by the Kubernetes system
    * `kube-public` This namespace is created automatically and is readable by all users (including those not authenticated). This namespace is mostly reserved for cluster usage, in case that some resources should be visible and readable publicly throughout the whole cluster. The public aspect of this namespace is only a convention, not a requirement.
-
+   * `kube-node-lease` This namespace for the lease objects associated with each node which improves the performance of the node heartbeats as the cluster scales.
```
Comment (wojtek-t) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md:66]: nit: This namespace is ...

Comment (kbhawkey) [reply content/en/docs/concepts/overview/working-with-objects/namespaces.md:66]: Thanks for the review!
Let's move this forward and create a follow-up PR to address the comment.

**Pull request merged**